spec/features/item_statistics_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'rails_helper'
+
+describe 'as any user' do
+  before(:each) do
+    @item_1 = create(:item) #add a times loop to build these?
+    @item_2 = create(:item)
+    @item_3 = create(:item)
+    @item_4 = create(:item)
+    @item_5 = create(:item)
+    @item_6 = create(:item)
+    @item_7 = create(:item)
+    @order_item_1 = create(:fulfilled_order_item, item: @item_1) #add a times loop to build these?
+    @order_item_2 = create(:fulfilled_order_item, item: @item_1)
+    @order_item_3 = create(:fulfilled_order_item, item: @item_1)
+    @order_item_3 = create(:fulfilled_order_item, item: @item_1)
```

Comment:
You've got the same variable name here on a bunch of these! Looks like the variable name doesn't really matter for this, so you could just go ahead and get rid of them entirely if you wanted.

Reply:
Face palm. Thank you for catching this.